utils/githooks/install

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+
+githook_root="${0%/*}"
+repo_root="${githook_root}/../../"
+
+pushd $repo_root > /dev/null
+for d in ${githook_root}/*.d; do
```

Comment:
(lint) Quote expansions in this for loop glob to prevent wordsplitting, e.g. "$dir"/*.txt . [SC2231]

Reply:
You should fix these.  You can run shellcheck on the script locally

Reply:
How? (also, probably a candidate for a future githook)

Reply:
Very O/S dependent of course, but on Linux, I just install the ShellCheck RPM.  VS Code then uses it to show linting issues as you edit.

But yes, please do fix these.